src/ui/layout/index.md

```diff
@@ -188,6 +188,71 @@ class MyApp extends StatelessWidget {
   or you can build your own set of custom widgets.
 {{site.alert.end}}
 
+#### Cupertino apps
+
+To create a `Cupertino` app, use a [`CupertinoPageScaffold`][] widget.
+
+Unlike `Material`, it doesn't provide a default banner or background color.
+You need to set these yourself.
+
+* To set default colors, you add a [`CupertinoThemeData`][].
+* To add a [`CupertinoNavigationBar`][] as a `navigationBar` property value.
+  You can modify per widget colors using [`CupertinoColors`][].
+
+* To layout the body of your app, set a `child` property with the
+  desired widget as its value, like `Center` or `Column`.
```

Comment:
Since there can only be one child perhaps "the" is better. Also the child of what? The app, the scaffold, the theme?

```suggestion
* To layout the body of your app, set the `child` property of your scaffold
  with the desired widget as its value, like `Center` or `Column`.
```